### PR TITLE
*: update anchor links to github style

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR updates all anchors to the Github style.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1515

### Which version does your change affect? 

`dev`, `v2.1`, `v3.0`

### CODE REVIEW

File: dev/TOC.md [hunk 1]
```diff
@@ -88,7 +88,7 @@
         - [使用限制](/dev/reference/tools/data-migration/overview.md#使用限制)
       + 核心特性
         - [Table Routing](/dev/reference/tools/data-migration/features/overview.md#table-routing)
-        - [Black & White Lists](/dev/reference/tools/data-migration/features/overview.md#black-white-table-lists)
+        - [Black & White Lists](/dev/reference/tools/data-migration/features/overview.md#black--white-table-lists)
```
Comment (lilin90) [inline dev/TOC.md]: This link path doesn't work on the testing page. PTAL @YiniXu9506

Comment (lilin90) [reply dev/TOC.md]: Fixed now.

**Comment by lilin90** [comment]
@yikeke Please help review this PR. Thanks!

File: dev/reference/tools/data-migration/configure/task-configuration-file.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ syncers:                                            # Syncer 处理单元运行
     max-retry: 100                                  # Syncer 同步到下游数据库出错的事务的重试次数（仅限于 DML 操作）。
 ```
 
-## 示例配置
+## 实例配置
```
Comment (lilin90) [inline dev/reference/tools/data-migration/configure/task-configuration-file.md]: Good catch!

**Comment by YiniXu9506** [comment]
LGTM

File: dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ TiDB Operator 使用 Helm 部署和管理 TiDB 集群。通过 Helm 获取的配
 | `pd.resources.requests.cpu` | 每个 PD Pod 的 CPU 资源请求 | `nil` |
 | `pd.resources.requests.memory` | 每个 PD Pod 的内存资源请求 | `nil` |
 | `pd.resources.requests.storage` | 每个 PD Pod 的存储容量请求 | `1Gi` |
-| `pd.affinity` | `pd.affinity` 定义 PD 的调度规则和偏好，详细请参考：[affinity-and-anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-baffinity) | `{}` |
+| `pd.affinity` | `pd.affinity` 定义 PD 的调度规则和偏好，详细请参考：[affinity-and-anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-affinity) | `{}` |
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]: Sharp eyes!

File: v3.0/tidb-in-kubernetes/reference/configuration/tidb-cluster.md [hunk 4]
```diff
@@ -134,7 +134,7 @@ TiDB 是分布式数据库，它的容灾需要做到在任一个物理拓扑节
 
 TiDB 服务容灾本质上基于 Kubernetes 的调度功能来实现的，为了优化调度，TiDB Operator 提供了自定义的调度器，该调度器通过指定的调度算法能在 host 层面，保证 TiDB 服务的容灾，而且目前 TiDB Cluster 使用该调度器作为默认调度器，设置项是上述列表中的 `schedulerName` 配置项。
 
-其它层面的容灾（例如 rack，zone，region）是通过 Affinity 的 `PodAntiAffinity` 来保证，通过 `PodAntiAffinity` 能尽量避免同一组件的不同实例部署到同一个物理拓扑节点上，从而达到容灾的目的，Affinity 的使用参考：[Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-baffinity) 。
+其它层面的容灾（例如 rack，zone，region）是通过 Affinity 的 `PodAntiAffinity` 来保证，通过 `PodAntiAffinity` 能尽量避免同一组件的不同实例部署到同一个物理拓扑节点上，从而达到容灾的目的，Affinity 的使用参考：[Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-affinity) 。
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]:
```suggestion
其它层面的容灾（例如 rack，zone，region）是通过 Affinity 的 `PodAntiAffinity` 来保证，通过 `PodAntiAffinity` 能尽量避免同一组件的不同实例部署到同一个物理拓扑节点上，从而达到容灾的目的，Affinity 的使用参考：[Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-affinity)。
```

File: dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md [hunk 5]
```diff
@@ -134,7 +134,7 @@ TiDB 是分布式数据库，它的容灾需要做到在任一个物理拓扑节
 
 TiDB 服务容灾本质上基于 Kubernetes 的调度功能来实现的，为了优化调度，TiDB Operator 提供了自定义的调度器，该调度器通过指定的调度算法能在 host 层面，保证 TiDB 服务的容灾，而且目前 TiDB Cluster 使用该调度器作为默认调度器，设置项是上述列表中的 `schedulerName` 配置项。
 
-其它层面的容灾（例如 rack，zone，region）是通过 Affinity 的 `PodAntiAffinity` 来保证，通过 `PodAntiAffinity` 能尽量避免同一组件的不同实例部署到同一个物理拓扑节点上，从而达到容灾的目的，Affinity 的使用参考：[Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-baffinity) 。
+其它层面的容灾（例如 rack，zone，region）是通过 Affinity 的 `PodAntiAffinity` 来保证，通过 `PodAntiAffinity` 能尽量避免同一组件的不同实例部署到同一个物理拓扑节点上，从而达到容灾的目的，Affinity 的使用参考：[Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-affinity) 。
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]:
```suggestion
其它层面的容灾（例如 rack，zone，region）是通过 Affinity 的 `PodAntiAffinity` 来保证，通过 `PodAntiAffinity` 能尽量避免同一组件的不同实例部署到同一个物理拓扑节点上，从而达到容灾的目的，Affinity 的使用参考：[Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#affinity-and-anti-affinity)。
```

**Comment by lilin90** [comment]
Rest LGTM

**Pull request merged**